### PR TITLE
Fixing some things in ansible-aliases

### PR DESCRIPTION
**Pull request by kopiro**

### CODE REVIEW

File: ansible-aliases.sh [hunk 1]
```diff
@@ -112,8 +112,10 @@ for _FILE in ${SHELL_CONFIG_FILES[@]}; do
 		else
 			if grep -q "${FILE_INCLUDE}" ${_FILE}; then
 				message "Aliases file is already installed in ${_FILE}"
+				exit 0
```
Comment (kopiro) [inline ansible-aliases.sh]: If aliases are already installed, no need to output commands again.

File: ansible-aliases.sh [hunk 2]
```diff
@@ -112,8 +112,10 @@ for _FILE in ${SHELL_CONFIG_FILES[@]}; do
...
 			else
 				cp ${_FILE} ${_FILE}.old
+				echo '' >> ${_FILE}
```
Comment (kopiro) [inline ansible-aliases.sh]: Relevant bug!!!
When file didn't have EOF at the end, this command broke everything.
Ensure EOF.

File: ansible-aliases.sh [hunk 3]
```diff
@@ -139,6 +141,4 @@ fi
 for _COMMAND in ${COMMANDS[@]}; do
 	echo "alias ${_COMMAND}='docker run ${SSH_KEY_ENV} ${PLAYBOOK_ENV} ${EXTRA_ENV} -it ${DOCKER_IMAGE}:${DOCKER_TAG} ${_COMMAND}'" >> ${FILE_NAME}
 	message "Alias ${_COMMAND} installed"
-done
-
-exit 0
```
Comment (kopiro) [inline ansible-aliases.sh]: Useless.

File: ansible-aliases.sh (hunk 1 above)
Comment (gadiener) [reply ansible-aliases.sh]: Although the aliases are already defined in the current shell, you may need to configure them with other parameters. So, you do not have to exit at that moment.

File: ansible-aliases.sh (hunk 2 above)
Comment (gadiener) [reply ansible-aliases.sh]: Thanks

**Review by gadiener — CHANGES_REQUESTED** [review]

File: ansible-aliases.sh (hunk 1 above)
Comment (kopiro) [reply ansible-aliases.sh]: Fine

**Review by gadiener — APPROVED** [review]

**Pull request merged**